### PR TITLE
drop crio.shutdown support

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind design

#### What this PR does / why we need it:
The interaction between crio.shutdown and crio wipe has always been kind of confusing. As we move towards internal wipe being the default (set in [this](github.com/cri-o/cri-o/pull/5218/) PR), it removes the need for crio.shutdown. Now, detecting whether a shutdown has happened happens when containers are restored, and no external files are consulted
#### Does this PR introduce a user-facing change?

```release-note
Drop support for the crio.shutdown.
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM
but would like @mrunalp to take a looksee

File: server/server.go [hunk 1]
```diff
@@ -295,26 +294,8 @@ func (s *Server) restore(ctx context.Context) []string {
 	return imagesOfDeletedContainers
 }
 
-// cleanupSandboxesOnShutdown Remove all running Sandboxes on system shutdown
-func (s *Server) cleanupSandboxesOnShutdown(ctx context.Context) {
-	_, err := os.Stat(shutdownFile)
-	if err == nil || !os.IsNotExist(err) {
-		log.Debugf(ctx, "Shutting down all sandboxes, on shutdown")
-		s.stopAllPodSandboxes(ctx)
```
Comment (saschagrunert) [inline server/server.go:303]: Isn't this function unused after that and can be removed as well?

**Comment by haircommander** [comment]
good point! fixed

File: server/server.go (hunk 1 above)
Comment (haircommander) [reply server/server.go:303]: good point! fixed

**Review by mrunalp — APPROVED** [review]
/lgtm

**Pull request merged**